.github/workflows/aft.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Dart
         uses: dart-lang/setup-dart@196f54580e9eee2797c57e85e289339f85e6779d # main
         with:
-          sdk: stable
+          sdk: beta
```

Comment:
This will switch back to stable when dart 3 becomes stable?

Reply:
Yes